src/middleware/AuthMiddleware.ts

```diff
@@ -1,35 +1,34 @@
-import {NextFunction, Request} from "express";
-import {Forbidden, Unauthorized} from "http-errors";
+import { NextFunction, Request } from "express";
+import { Forbidden, Unauthorized } from "http-errors";
 import JWTs from "../utils/JWTs";
-import {asyncHandler} from "../utils/AsyncHandler";
-import {JwtPayload, TokenExpiredError} from "jsonwebtoken";
-import {AdminService} from "../services/AdminService";
+import { asyncHandler } from "../utils/AsyncHandler";
+import { JwtPayload } from "jsonwebtoken";
+import { AdminService } from "../services/AdminService";
 
-const adminService = new AdminService()
+const adminService = new AdminService();
 
-export const isAdmin = asyncHandler(async (req: Request<{}, {}, {}>, res, next: NextFunction) => {
-  let jwt = req.headers.authorization
-  let decodedJwt: JwtPayload
+export const isAdmin = asyncHandler(
+  async (req: Request<{}, {}, {}>, res, next: NextFunction) => {
+    let jwt = req.headers.authorization;
+    let decodedJwt: JwtPayload;
 
-  if (!jwt)
-    throw new Unauthorized("AccessToken missing")
+    if (!jwt) throw new Unauthorized("AccessToken missing");
 
-  if (jwt.toLowerCase().startsWith('bearer'))
-    jwt = jwt.slice('bearer'.length).trim();
+    if (jwt.toLowerCase().startsWith("bearer"))
+      jwt = jwt.slice("bearer".length).trim();
 
-  try {
-    decodedJwt = JWTs.verifyAndDecodeAccessToken(jwt)
-  } catch (e) {
-    if (e instanceof TokenExpiredError) {
-      throw new Unauthorized("AccessToken expired")
-    } else {
-      throw new Unauthorized("AccessToken invalid")
+    try {
+      decodedJwt = JWTs.verifyAndDecodeAccessToken(jwt);
+    } catch (e) {
+      throw new Forbidden(e.message);
     }
```

Comment:
Why remove the more precise feedback.
If an AccessToken is `invalid`, something crucial may be broken, while `expired` is totally normal and intended.

Reply:
You can see various errors on the jsonwebtokens npm page. Throwing one error with custom error message might be more precise than "AccessToken expired" or "AccessToken invalid"